File: RA1/clase5/Ejercicio1.java
import java.util.Scanner;

class Ejercicio1{

    static int potencia(int base, int exp){
        int pot=1;
        for(; exp>0; exp--)
            pot *= base;
        return pot;
    }

    public static void main(String[] args){
        Scanner lector = new Scanner(System.in);
        System.out.println("Ingrese la base: ");
        int b = lector.nextInt();
        System.out.println("Ingrese el exponente: ");
        int exp = lector.nextInt();

        int resultado = potencia(b, exp);

        System.out.println("La potencia es: " + resultado);
    }
}

File: RA1/clase6/ejercicio_1.py
def potencia(b:int, e:int) -> int:
    return b**e

if __name__ == "__main__":
    base = int(input("Ingrese el valor de la base: "))
    exp = int(input("ingrese el valor del exponente: "))

    resultado = potencia(base, exp)

    print("El resultado de la potencia de " + str(base) + " exponente " + str(exp) + " es " + str(resultado))